app/logic/fileHandler.py

```diff
@@ -15,11 +15,17 @@ def __init__(self,files=None, courseId=None, eventId=None):
             # eventID is not included in the path, because it is now a part of the attachment filename.
             self.path = os.path.join(self.path, app.config['files']['event_attachment_path'])
         
+    def makeDirectory(self):
+        # Creating the directory when call in saveFiles
```

Comment:
Don't assume where this function will be called.

Reply:
Also there should be a better description that describes what the function does and/or why it exists